docs/user-guides/server_config_guide.md

```diff
@@ -60,16 +60,25 @@ Check and adjust your:
 
 ## Main configuration file
 
-<!-- List (&explain) the most important options -->
+The main server configuration file is called `murmur.ini` (it can be renamed though).
+
+You should find an example murmur.ini file alongside your installation, but you can also find the most recent **example file (with explanations) on [Github](https://github.com/mumble-voip/mumble/blob/master/scripts/murmur.ini)**.
+
+Below is a list of the most important options:
+
+`welcometext="Welcome on our Server!"` - the text will be send to every new user via chatmessage; if it's empty, no message will be send
+
+`port=64738` - sets the port (see above); `64738` is the standard port
+
+`serverpassword=` - sets the Serverpassword; if it's empty, users can connect without a password.
+
+`bandwidth=72000` - sets the maximum bandwidth per user (in bits per second); setting it higher can increase the audio quality
+
+`users=100` - sets the maximum number of online users
 
-<!-- welcometext=
-port=
-serverpassword=
-bandwidth=
-users=
-registerName= -->
+`registerName=` - sets the Servername and the name of the main channel
```

Comment:
We could consider using a table instead of a list for these :point_up:

Reply:
Well, I can do that.
I only thought that it's not necessary for so few config keys.